core/src/cats/postgresql.cc

```diff
@@ -800,6 +815,7 @@ uint64_t BareosDbPostgresql::SqlInsertAutokeyRecord(const char* query,
 
 void BareosDbPostgresql::SqlUpdateField(int i)
 {
+  AssertOwnership();
```

Comment:
`SqlUpdateField()` is only called by `SqlFetchField()`, so this check is redundant.